app/controllers/pseuds_controller.rb

```diff
@@ -146,4 +146,14 @@ def destroy
 
     redirect_to(user_pseuds_url(@user))
   end
+
+  private
+
+  def pseud_params
+    params.require(:pseud).permit(
+      :user_id, :name, :description, :is_default, :icon, :delete_icon, :icon_file_name,
```

Comment:
You are completely correct. I must of been thinking about the icon_file_name. sorry ( have to look at how it works now )